src/main/java/com/powsybl/openloadflow/network/impl/LfHvdcImpl.java

```diff
@@ -52,6 +54,7 @@ public LfHvdcImpl(String id, LfBus bus1, LfBus bus2, LfNetwork network, HvdcLine
         this.id = Objects.requireNonNull(id);
         this.bus1 = bus1;
         this.bus2 = bus2;
+        this.r = hvdcLine.getR() / PerUnit.zb(hvdcLine.getNominalV()); // Storing R directly in per unit
```

Comment:
Can you remain consistent with other treatment. P0 is not "stored" in per-unit.

Reply:
Yes, we had this question of consistency with two possible options :
- Being consistent with the other treatments so introducing an additional property nominalV in LfHvdc
- Changing the other treatment to "store" directly p0 and droop in per unit (but it may need some other modifications and maybe another PR)

I don't know which one is the best, but the easiest seems to be the first option